### ISSUE TITLE
Incorrect regen values for veterancy

### ISSUE THREAD

**Issue by Exotic-Retard** [spooky/unitdb#88]
not too long ago, the veterancy system was changed. as a result the veterancy stats are determined by script and not so much by blueprint.

![](https://i.imgur.com/HWccRB2.png)
here the uef acu having the incorrect values;
1.veterancy is mass based so by default, a unit gets a veterancy every time it reaches 200% of its mass cost in killed units, which is an exception for the ACUs, and experimentals, they need 50% (1000m for uef acu)
this is determined by the VeteranRequirementMult value in the blueprint:
vet requirement = unit mass cost * (VeteranRequirementMult or 2)
for uef acu: 1000 = 2000 * 0.5

2. regen is applied per vet per "tech level"
t1 - t2 - t3 - sacu - t4
1 - 3 - 6 - 12 - 24(?)
naval units get vet as if they were a category higher, so a t1 frigate gets 3 regen per vet.
acus count as t3 units in terms of vet regen. probably.

these stats need to be checked really. so DONT use the numbers given here!

also, there are around 10 different rules on top of this which make the system impossible to predict exactly when you get a vet, so just these changes are good enough really.

**Comment by spooky** [comment]
yeah.. vet has been moved away from the .bp and into .lua... I guess I'll just remove that cause calculating this in the js is not possible without accessing some additional files and replicating the same logic that's in lua... and ... .that'll get outdated pretty soon...
